cmd/availability-prober/main.go

```diff
@@ -0,0 +1,56 @@
+package main
```

Comment:
@alvaroaleman just one more nit, can you move this to the root of the package to the root of the repo?
So far we only have CLI stuff (`hypershift` command) in the `cmd` package.

Reply:
updated